ui-grid/ui-grid.d.ts

```diff
@@ -3527,8 +3531,9 @@ declare module uiGrid {
      * which would typically be one of many column definitions within the
      * gridOptions.columnDefs array
      */
-    export interface IColumnDef extends cellNav.IColumnDef, edit.IColumnDef, exporter.IColumnDef, grouping.IColumnDef,
-        moveColumns.IColumnDef, pinning.IColumnDef, resizeColumns.IColumnDef, treeBase.IColumnDef {
+    export type IColumnDef = IColumnDefOf<any>;
+    export interface IColumnDefOf<TEntity> extends cellNav.IColumnDef, edit.IColumnDef<TEntity>, exporter.IColumnDef, grouping.IColumnDef,
```

Comment:
are any of these lines over 120 chars?  I'm trying to keep to the convention of keeping lines shorter than that